lib/stat_tracker.rb

```diff
@@ -33,9 +33,27 @@ def total_goals_per_game
     end
   end
 
+  def total_away_wins
+    away_wins = 0
+    @games.each do |game|
+      if game.away_goals > game.home_goals
+        away_wins += 1
+      end
+    end
+    away_wins
+  end
+
 # Game Statistics Tests - Stat Methods #
   def highest_total_score
-    total_goals_per_game.max_by {|game_id, total_goals| total_goals}[1]
+    total_goals_per_game.max_by {|game_id, total_goals| total_goals}
+  end
+
+  def lowest_total_score
+    total_goals_per_game.min_by {|game_id, total_goals| total_goals}
+  end
```

Comment:
This one is also missing the [1]. This method is currently returning an array of [gameid, score]. I think it passed the test because there was an assert instead of an assert_equal

Reply:
Yeah, I think that was an error I made while merging.